static/app/utils/theme.tsx

```diff
@@ -484,7 +484,7 @@ const generateBadgeTheme = (colors: BaseColors) => ({
     color: colors.white,
   },
   experimental: {
-    background: `linear-gradient(90deg, ${colors.blue300}, ${colors.green300})`,
+    background: `linear-gradient(270deg, #F5BE50 0%, #E06581 100%)`,
```

Comment:
these colors are not part of our theme...shall we add them @Jesse-Box ?

Reply:
Nah I would just use whatever badge variant you used previously.

Reply:
oh but these colors are so cool! :sad

Reply:
tbh, I just don't think it's important enough to extend the theme.